### PR TITLE
added electron build cmd

### PR DESCRIPTION
**Pull request by marionleborgne**
@lscheinkman @brev 

it's now possible to build the OSX automatically app by running 

```
npm install
npm run build 
```
This will generate `unicorn.app` in `unicorn-darwin-x64`.

I am still working on the app signing and packaging the model runner with pyinstaller (UNI-165). but this PR should at least take care of the UNI-166 task. 

cc @jcasner

### CODE REVIEW

File: unicorn/package.json [hunk 1]
```diff
@@ -85,7 +86,8 @@
     "babel-plugin-typecheck": "1.2.0",
     "casperjs": "1.1.0-beta3",
     "electron-debug": "0.2.0",
-    "electron-prebuilt": "0.31.2",
+    "electron-packager": "^5.1.0",
+    "electron-prebuilt": "^0.32.3",
```
Comment (lscheinkman) [inline unicorn/package.json]: We should use absolute version numbers for dependencies

File: unicorn/frontend/main.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ app.on('ready', () => {
     // @TODO fill out options
     //  https://github.com/atom/electron/blob/master/docs/api/browser-window.md
   });
-  mainWindow.loadUrl('file://' + __dirname + config.get('entryHtml'));
+  mainWindow.loadUrl('file://' + __dirname + '/browser/index.html');
```
Comment (lscheinkman) [inline unicorn/frontend/main.js]: @marionleborgne We can leave this value hardcoded for now, but we will need to use the `ConfigServer` in the future.
Maybe @brev can take a look at the `ConfigServer` and try to understand why `config.get('entryHtml')` returns `null` here

**Comment by brev** [comment]
sorry for the delay @marionleborgne @lscheinkman - besides this one, I'm all caught up now. I'll take a look first thing Mon morning.

**Comment by marionleborgne** [comment]
@brev did you have a chance to look at the `ConfigServer`and why `config.get('entryHtml')` returns `null`?

**Comment by brev** [comment]
:+1: very cool, works great for me, thanks. Looking at the config setting now, been blocked by Avik work still.

**Comment by brev** [comment]
@marionleborgne will you please add some docs to README.md on the 'npm run build' command? thanks

File: unicorn/frontend/main.js (hunk 2 above)
Comment (brev) [inline unicorn/frontend/main.js]: @marionleborgne @lscheinkman Path problem it looks like, trying to get details

Comment (brev) [inline unicorn/frontend/main.js]: I don't think this is blocking merge btw

**Comment by marionleborgne** [comment]
@brev yes i updated the readme

File: unicorn/frontend/main.js (hunk 2 above)
Comment (brev) [inline unicorn/frontend/main.js]: @marionleborgne @lscheinkman might be a few days before I get to this, FYI. sorry!

**Pull request merged**